migrations/emailmonitor/oxa_email_monitor.py

```diff
@@ -0,0 +1,179 @@
+import MySQLdb
+import collections
+import os
+import re
+from datetime import datetime,timedelta
+
+
+def CreateYearMonthDay():
```

Comment:
Not blocking this PR since this is just style but it would be great to adhere to python style best practices.

(e.g. All function and variable names should be snake case: def CreateYearMonthDay(): => def create_year_month_day():)